packages/venia-concept/src/components/Navigation/navigation.js

```diff
@@ -140,17 +143,46 @@ class Navigation extends PureComponent {
         this.showCreateAccountForm();
     };
 
+    forgotPassword = () => {};
+
+    setForgotPasswordForm = () => {
```

Comment:
Why are we waiting for `SignIn`'s `onForgotPassword`to set `this.forgotPassword`?

In other words, can we just

```js
forgotPassword = className => {
  return (
    <div className={className}>
      <ForgotPassword
        initialValues={{ email: this.state.defaultUsername }}
        onClose={this.closeForgotPassword}
      />
    </div>
  );
}
```

?

Reply:
There is a bug with informed, it uses initialValues only when component is mounted. There is a comment in this file above.

Reply:
Yeah, I can add a comment.